site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -145,6 +145,69 @@
     <br/>
     <hr>
 
+    <div class="dropdown-wrapper">
+        <div id="tomorrow-table-dropdown" onclick="updateDropdownStates($(this).find('.dropdown-arrow').hasClass('down'), 'tomorrow_polls_dropdown', '{{ base_url }}')" class="dropdown-bar">
+            <h2> Tomorrow's Polls </h2>
+            <div class="dropdown-btn">
+                <i class="dropdown-arrow{{dropdown_states['future'] ? ' down' : ' right'}} fa fa-chevron-right"></i>
+            </div>
+        </div>
+        <table id="tomorrow-table" class="table table-striped" style="width:100%;{{ dropdown_states['future'] ? '' : ' display: none;'}}">
+            <col style="width: 5%">
+            <col style="width: 5%">
+            <col style="width: 50%">
+            <col style="width: 15%">
+            <col style="width: 10%">
+            <col style="width: 15%">
+            <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col">Date Released</th>
+                    <th scope="col">Responses</th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for poll in future_polls %}
+                    {% if poll.getReleaseDate() | date("Y-m-d") == ('now'|date_modify("+1 day")|date('Y-m-d')) %}
+                    <tr>
+                        <td>
+                                <a class="fas fa-pencil-alt black-btn" tabindex="0" href="{{base_url}}/editPoll/{{ poll.getId() }}" title="edit poll {{ poll.getName() }}" aria-label="edit poll {{ poll.getId() }}">
+                                        <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
+                                </a>
+                        </td>
+                        <td>
+                                <a class="fas fa-trash black-btn" tabindex="0" href='javascript:newDeletePollForm("{{ poll.getId() }}", "{{ poll.getName() }}", "{{ base_url }}");' title="delete poll {{ poll.getName() }}" aria-label="delete poll {{ poll.getId() }}"></a>
+                        </td>
+                        <td style="text-align: left;">
+                            {{ poll.getName() }}
+                        </td>
+                        <td> {{ poll.getReleaseDate() | date("Y-m-d") }} </td>
+
+                        <td id="poll_{{ poll.getId() }}_responses"> {{ poll.getUserResponses()|length }} </td>
```

Comment:
This line is the source of the failing tests.  As the loop containing this code iterates over each poll, this line will attempt to load all of the user responses to each poll, and then count them.  This is a bad approach, because it creates a separate query each time the loop iterates, and gathers unnecessary data from the database anyway.  A better option would be to mimic the same line in the "today's polls" section by only querying the number of answers to each poll.

Reply:
To reproduce the duplicate query warning, you just have to put multiple tests in the "tomorrow's polls" section.